<launch>
    <!-- Set parameters -->
    <param name="graphml_file" value="$(find control)/config/Competition_track_graph.graphml"/>
    <param name="key_nodes_file" value="$(find control)/config/global_path_key_nodes.yaml"/>
    <param name="publish_rate" value="1.0"/>

    <!-- Launch the Python node -->
    <node pkg="control"
          type="node_to_path.py"
          name="node_to_path"
          output="screen">
    </node>

</launch>
